client/app/components/admin/Layout.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tabs from 'antd/lib/tabs';
+import { PageHeader } from '@/components/PageHeader';
+
+import './layout.less';
+
+export default function Layout({ activeTab, children }) {
+  return (
+    <div className="container admin-page-layout">
+      <PageHeader title="Admin" />
```

Comment:
ux nit: Shouldn't this be titled "System Status"?

Reply:
No, "System Status" is the first tab